app/config/question-bank.js

```diff
@@ -224,18 +224,10 @@ const questionBank = {
               elseUrl: 'project-subject'
             }
           },
-          // if user selects Solar technologies system on project-subject page, redirect to /country, otherwise redirect to /planning-permission
-          dependantNextUrl: {
-            dependentQuestionYarKey: 'projectSubject',
-            dependentAnswerKeysArray: ['project-subject-A2'],
-            urlOptions: {
-              thenUrl: 'country',
-              elseUrl: 'planning-permission'
-            }
-          },
           url: 'legal-status',
           baseUrl: 'legal-status',
-          // preValidationKeys: ['projectSubject'],
+          nextUrl: 'country',
+          preValidationKeys: ['businessLocation'],
```

Comment:
Can you comment this out for now pls? theres 3 different pages that can lead to legal-status (as far as i can see on mural)

Also can you update the back urls to include the 3 back destinations? (**project-type** if user chose **solar**, **who are you** if user chose **farmer** and **is business in england** if user was a contractor)

we have url switching with 3 routes in slurry/ahw but not in robotics yet, so may need to do a wee update to urls.js